dissect/target/tools/shell.py

```diff
@@ -504,17 +511,26 @@ def cmd_ls(self, args, stdout):
     def print_extensive_file_stat(self, stdout, target_path: fsutil.TargetPath, name: str) -> None:
         """Print the file status."""
         entry = target_path.get()
+        symlink = ""
 
         # Entry is an NTFSEntry and has alternative datastream
         if hasattr(entry, "deref") and ":" in name:
             # retrieve the ads from the NTFS entry
             ads_path = fsutil.join(fsutil.dirname(entry.path, separator=entry.fs.alt_separator), name)
             entry = entry.fs.get(ads_path)
 
-        stat = entry.lstat()
+        try:
+            stat = entry.stat()
+        except FileNotFoundError:  # Incase symlinks cant be found, due to them not being picked up by acquire
+            stat = entry.lstat()
```

Comment:
Shouldn't this remain `lstat`? The `ls` command doesn't show the stat information of the target but rather of the symlink file itself.

Reply:
Yep! Going to revert it back to just `entry.lstat()`